chapters/custom-license-namespaces.md

```diff
@@ -0,0 +1,194 @@
+
+A valid namespace using the DNS name format
+is a standard order DNS name,
+such as `example.com`.
+This DNS name
+must only use characters permitted by the license expression syntax
+(i.e., letters, digits, "-" and ".").
+It must not contain a period followed by a hyphen,
+as that denotes the end of the namespace.
+
+Each custom license identifier under this namespace
+consists of the following, in order:
+1. the prefix `LicenseRef-`
+2. a period
+3. the namespace
+4. a period and a hyphen
+5. any other characters permitted by the license expression syntax (i.e., letters, digits, "-" and ".")
+
+For example, for the `example.com` namespace,
+the following are identifiers:
+* `LicenseRef-.example.com.-ABC-1.0`
+* `LicenseRef-.example.com.-EULA-v3.1`
+* `LicenseRef-.example.com.-anything`
+
+## Z.3.2 Organization name format
+
+A valid namespace using the organization name format
+is the organization's name,
+such as `ExampleCorp`.
+This organization name
+must only use characters permitted by the license expression syntax
+(i.e., letters, digits, "-" and ".").
+It must not contain a double hyphen,
+as that denotes the end of the namespace.
+
+Each custom license identifier under this namespace
+consists of the following, in order:
+1. the prefix `LicenseRef-`
+2. a hyphen
+3. the namespace
+4. two hyphens
+5. any other characters permitted by the license expression syntax (i.e., letters, digits, "-" and ".")
+
+For example, for the `ExampleCorp` namespace, the following could be valid identifiers:
+* `LicenseRef--ExampleCorp--ABC-1.0`
+* `LicenseRef--ExampleCorp--EULA-v3.1`
+* `LicenseRef--ExampleCorp--anything`
+
+
+## Z.5 License definitions
+
+All the licenses in a namespace
+have to be defined in an SPDX document.
+This annex does not provide any additional mechanism
+for this task,
+as the standard
+[Other licensing information detected](./other-licensing-information-detected.md) sections
+of an SPDX document
+can be used without any special consideration.
+
+The SPDX document should not define
+any licenses with identifiers
+other than those in its own namespace.
+
+
+## Z.6 Namespace registration
+
+In order for a namespace to be usable
+in custom license identifiers,
+it must be registered with the SPDX project.
+The registration request must provide
+the namespace and
+the SPDX document defining the licenses.
+
+This registration will be completed
+using a mechanism to be provided by the SPDX project.
+The exact details of the mechanism
+for registration requests
+will be made publicly available
+at a later date,
+but it may consist of submitting pull requests
+to a public repository
+or using another custom tool for this purpose.
+
+
+## Z.7 Caveats
+
+This format attempts to provide a stable mechanism
+for use of custom identifiers.
+However, it should be noted that the stability
+will always be subject to the namespace maintainer's control.
+In other words,
+although maintainers should not modify a defined license text
```

Comment:
I'm not certain how the standards-world all-caps words work, but should this be "SHOULD NOT" instead of "should not"?